src/utils/storage.ts

```diff
@@ -0,0 +1,21 @@
+import Todo from '../ui/components/-utils/todo';
+
+export default class StorageService {
```

Comment:
I suggest a more evocative name like `TodoStore` and the file to `todo-store.ts`.